packages/shared/src/modules/activities/libs/types/finish-section-dto.type.ts

```diff
@@ -0,0 +1,11 @@
+type FinishSectionDto = {
```

Comment:
```suggestion
type ActivityFinishSectionDto = {
```
do not forget about file name

add response or request